### DISCUSSION TITLE
Data about position of the beds, obstacles, and path planning/ navigation.

### DISCUSSION THREAD

**Discussion by SerValera** [larics/icuas24_competition#10] [General]
Good day!

We would like to inquire about obtaining data for path planning algorithms. Should we explore the given area using visual sensors, or can we obtain the positions of the beds from description or models files?
Should we solve navigation problem in this challenge? 

Best regard,
Valerii

**Comment by fpetric** [comment]
Hi @SerValera, 
you do not have to do exploration, you can obtain the positions of the beds from the models/description or whichever way is the most convenient for you. You can use gazebo/mavros position once we resolve the ambiguity (as pointed out in #7) as a localization, so you only need to plan and execute paths/trajectories.
